packages/shared/src/utils/validation/declarative-validation-rules.ts

```diff
@@ -64,7 +64,15 @@ export const validLength = (value = '', options: TOptions) =>
     (options.max ? value.length <= Number(options.max) : true);
 export const validPassword = (value: string) => /^(?=.*[a-z])(?=.*\d)(?=.*[A-Z])[!-~]{8,25}$/.test(value);
 export const validEmail = (value: string) => /^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,63}$/.test(value);
-const validBarrier = (value: string) => /^[+-]?\d+\.?\d*$/.test(value);
+const validBarrier = (value: string) => {
+    if (value === '+' || value === '-') {
+        return {
+            is_ok: false,
```

Comment:
Maybe is_valid? Or is_correct? is_ok is too familiar

Reply:
I haven't invented it, `is_ok` is what is returned across all other validation rules across this file 😄

Reply:
O_o

Reply:
The more I work here, the less I'm surprised ))))